controllers/AuthController.js

```diff
@@ -0,0 +1,26 @@
+import sha1 from 'sha1';
+import uuid4 from 'uuid';
```

Comment:
you are importing like in python. it should be import { v4 } from 'uuid' or import { v4 as uuid4} from 'uuid'

Reply:
I saw something similar, i actually thought that way would work. that my be where my errors are coming from